author/urls.py

```diff
@@ -1,3 +1,13 @@
-# Create your urls here
+from django.urls import path, include
+from rest_framework import routers
 
-urlpatterns = []
+from author.views import AuthorViewSet
+
+router = routers.DefaultRouter()
+router.register("authors", AuthorViewSet, basename="manage")
+
+urlpatterns = [
+    path("", include(router.urls)),
+]
```

Comment:
No need to use include(if basepath is not provided) instead use:
urlpatterns = router.urls